relayer/src/index.ts

```diff
@@ -23,6 +29,22 @@ const createApi = async (url: string, types?: RegistryTypes) => {
   return api;
 };
 
+// TODO: consider moving to utils
+const sleep = (ms: number): Promise<void> =>
+  new Promise((resolve) => setTimeout(resolve, ms));
+
+const createFeeds = async (target: Target): Promise<number[]> => {
+  const feedIds: number[] = [];
+
+  for await (const _ of config.sourceChainUrls) {
+    await target.createFeed(feedIds);
+  }
+
+  await sleep(15000);
```

Comment:
I think this needs a comment `// TODO: Replace sleep() with proper feed creation tracking`

Reply:
this is temporary - already removed it in the refactoring branch, will be in the next PR